.vscode/launch.json

```diff
@@ -0,0 +1,40 @@
+{
```

Comment:
Probably fine to commit these .vscode files other option is just to modify .gitignore 

Up to you, I'm fine with either

Reply:
Committing them should make it easier to run locally, it's intentional.